README.md

```diff
@@ -0,0 +1,62 @@
+# Installation
```

Comment:
I don't think it's a good fit for Flathub if you expect user to create systemd unit. Instructions won't also cover all possible options: what if user doesn't use systemd? Also README duplicates official flatpak docs.

Reply:
I wouldn't say I expect them to create a systemd unit. The "default" launching behavior here is to start it in interactive mode, which should cover the most common usage, which is stress-testing.
The interactive mode also allows it to be used for distributed computing but it's only in this case that the systemd unit becomes useful.
About the possible options for init systems, unfortunately I've got no idea/experience about the others. So far, all my systems have used systemd.